docs/website/static/js/home.js

```diff
@@ -38,16 +38,18 @@
     var n = n || 1
 
     schedule(function () {
-      var diff = top - page.scrollTop
+      var diff = top - window.pageYOffset
       var increment = scrollIncrement * n
+      var currentY = window.pageYOffset
```

Comment:
Move `currentY` definition above `diff` definition, then `var diff = top - currentY`

Reply:
@lphamilton once you address @mikol's suggestion, we can merge this.